Automattic/jetpack
# Improve accessibility and display of math fallback(Protect)

Fixes #6714 & [this display issue](https://twitter.com/WPThemeSpeed/status/1201695870317670400)

#### Changes proposed in this Pull Request:

- Moved math equation inside a label and tied it to input field using `for="jetpack_protect_answer"`
- Changed input field's type to number from text
- Increased input field's width from 30px to 45px to properly accommodate two digit number


**Does this pull request change what data or activity we track or use?**
No

#### Testing instructions:

1.) Checkout the code on a WP test site that you've admin access to

2.) Make sure "Protect" is enabled under WP-Admin (Jetpack -> Settings -> Security)

3.) Test login with a test user. Trigger the Math captcha by attempting a fixed number of failed login using the correct username, but with the wrong password. Or you can directly trigger it by changing the if clause to `true` here:
https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/modules/protect.php#L643

4.) Use a screen reader to test the reader reads out the math equation properly. Enter the right and wrong numbers to make sure the results are as intended.

**Current dom screenshot**: 
<img width="1508" alt="math_protect_old" src="https://user-images.githubusercontent.com/95693514/146587927-b340e24e-672a-4569-bb2a-3c137c62abec.png">

**Current page Apple VoiceOver recording**

https://user-images.githubusercontent.com/95693514/146588132-6b447b30-99f9-4194-b8e7-3ad0ea7cff61.mov



**New dom screenshot**:
<img width="1393" alt="math_protect_new" src="https://user-images.githubusercontent.com/95693514/146587962-f15efc44-1e2b-4e88-bb1d-c06c5bd4835b.png">

**New page Apple VoiceOver recording**

https://user-images.githubusercontent.com/95693514/146588218-65d66f46-6c91-4567-9804-c9113d154c99.mov
